platform-includes/configuration/auto-session-tracking/flutter.mdx

```diff
@@ -1,29 +1,23 @@
 To benefit from the health data, you must use at least version 4.0.0 of the Flutter SDK.
 
-By default, the session is terminated once the application is in the background for more than 30 seconds. You can change the time out with the option named `sessionTrackingIntervalMillis`. It takes the amount in milliseconds. For example, to configure it to be 60 seconds:
+#### Session Timeout
 
-```dart
-import 'package:flutter/widgets.dart';
-import 'package:sentry_flutter/sentry_flutter.dart';
+By default, the session is terminated once the application is in the background for more than `30 seconds`. You can change the default timeout with the option `autoSessionTrackingInterval` to a `Duration` of your choosing.
 
-Future<void> main() async {
-  await SentryFlutter.init(
-    (options) => options.autoSessionTrackingInterval = const Duration(milliseconds: 60000)
-    appRunner: () => runApp(MyApp()),
-  );
-}
+```dart
+SentryFlutter.init((options) {
+  // Change timeout duration until session is terminated in background
+  options.autoSessionTrackingInterval = const Duration(seconds: 60)
+}, appRunner: () => runApp(MyApp()));
```

Comment:
I am unsure whether we should leave the appRunner in there, it is an optional parameter